app/src/main/java/ch/epfl/sdp/blindly/user/User.kt

```diff
@@ -17,14 +17,14 @@ import java.time.Period
 @Serializable
 class User private constructor(
     val username: String?,
-    val location: String?,
+    val location: List<Double>?,
     val birthday: String?,
     val gender: String?,
     val sexualOrientations: List<String>,
     val showMe: String?,
     val passions: List<String>,
     val radius: Int?,
-    val matches: List<User>,
+    val matches: List<String>,
```

Comment:
It is not very clear, why is the changed? Is `matches` now a list of IDs? If it is the case, I think renaming to something like matchesIds could be clearer.

Reply:
It was actually discussed in the meetings, this is preferred not to unnecessarily store whole user for each match where as our matching algorithm only needs UIDs which are stored as Strings.